plugins/main/public/components/settings/settings.tsx

```diff
@@ -299,8 +299,6 @@ class SettingsComponent extends React.Component {
       };
 
       this.setState({ load: false });
-      // TODO: this seems not to be used to display or not the index pattern selector
-      AppState.setPatternSelector(this.props.configurationIPSelector);
       const pattern = AppState.getCurrentPattern();
```

Comment:
'pattern' is declared but its value is never read

Reply:
This should be reviewed using the linter when this is included and the related pull request check.